pkgs/top-level/all-packages.nix

```diff
@@ -24189,11 +24189,10 @@ with pkgs;
   nv-codec-headers-12 = callPackage ../development/libraries/nv-codec-headers/12_x.nix { };
 
   nvidiaCtkPackages =
-    callPackage ../applications/virtualization/nvidia-container-toolkit/packages.nix
+    callPackage ../by-name/nv/nvidia-container-toolkit/packages.nix
       { };
   inherit (nvidiaCtkPackages)
     nvidia-docker
-    nvidia-podman
```

Comment:
Removing this because podman supports CDI since 3.2.0.

Reply:
To be clear, I only suggested to keep `libnvidia-container`, not `nvidia-{docker,podman}`

Reply:
I think it's ok to remove those two, and we should also update the release notes

Reply:
That would mean breaking support for docker_20_10 GPU support. This is why I'm keeping it.

Reply:
I see 👍🏻